Revert "Fix default path of `VMM_KO`."

fearedbliss opened the pull request:
This reverts commit f7ac871e5e965754c0db5074828b1ad9c8fd756d. Due to it breaking the workaround spoken about in issue 31. Actually, reverting this allowed the normal resume process to work it seems (meaning a normal restart vmm and no additional logic actually worked).

https://github.com/pgj/freebsd-wifibox/issues/31

pgj commented:
This is not a real revert commit as it includes additional changes.  On the other hand, I am not yet convinced if that specific change needs to be backed out.  It had its own reason, it fixed other problems, see #29.  Note that it also refers to a configurable `make` which the user is free to change on build.

fearedbliss commented:
That's correct, I added the additional fixed as mentioned in the main thread, and I also updated the comment in the actual commit message, however the GitHub one didn't get updated and still states just revert. That should also match the commit message to indicates that it's not just the original revert message.

All I can do is report to you what I've found and what ended up fixing it after extensive testing and bisecting multiple repos. So there is definitely a regression as part of that VMM commit.

pgj commented:
I think you should be able to change the PR title, but let me know if that is not the case.

fearedbliss commented:
Yea I can change it (will do now) but just stating what happened.

pgj commented:
We should understand why the default location of `VMM_KO` makes this difference.  The idea behind that change to avoid reloading `vmm.ko` which could disturb the system -- but not doing it should not cause any trouble.

pgj commented:
Maybe in your case that is actually required...  Similar to the one what I have been observing on FreeBSD 12.
